package.json

```diff
@@ -1233,7 +1238,8 @@
                 "maxVariableRecurse": 1,
                 "maxStringLen": 64,
                 "maxArrayValues": 64,
-                "maxStructFields": -1
+                "maxStructFields": -1,
+                "showGlobalVariables": false
```

Comment:
I believe we need to pull `showGlobalVariables` one level up.
The properties in LoadConfig is a 1:1 to what is at https://github.com/go-delve/delve/blob/4c9a72e486f1f0d0c90ecede8415a871dced8117/service/api/conversions.go#L276